Sources/Subs.php

```diff
@@ -829,6 +829,9 @@ function timeformat($log_time, $show_today = true, $offset_type = false, $proces
 	if ($context['server']['is_windows'] && strpos($str, '%e') !== false)
 		$str = str_replace('%e', ltrim(strftime('%d', $time), '0'), $str);
 
+	// fix unkown %l with %I
```

Comment:
`%l` is part of the strftrime format specifications. The German version of the strftrime documentation is incomplete. See the English version for a description of `%l` in comparison to `%I`: http://php.net/manual/en/function.strftime.php